scipy/stats/_continuous_distns.py

```diff
@@ -2900,6 +2900,186 @@ def _entropy(self, c):
 genhalflogistic = genhalflogistic_gen(a=0.0, name='genhalflogistic')
 
 
+class genhyperbolic_gen(rv_continuous):
+    r"""A generalized hyperbolic continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t, norminvgaus, geninvgauss, laplace, cauchy
```

Comment:
```suggestion
    t, norminvgauss, geninvgauss, laplace, cauchy
```
This might be the only thing causing the doc build failure.

Reply:
yes i confirm, i have that change already fixed locally, pushing it in short

Reply:
Seems like doc build fails because on azure introspection [yields 103 continuos distribution](https://dev.azure.com/scipy-org/SciPy/_build/results?buildId=10036&view=logs&j=4c925117-6155-59c4-749e-aa25fa289e87&t=8e226bf9-5f8b-5d2e-42ac-66bf72f60081&l=339 ), whereas when i check on my machine i get 102, as one would expect. I am not sure why this happens however

```
>>> dist_continuous = [d for d in dir(stats) if isinstance(getattr(stats, d), stats.rv_continuous)]
>>> len(dist_continuous)
102
>>>
```

Reply:
There are 102 in the current master. Adding this one would make it 103. Something must have merged since you last merged master. Go ahead and change it to 103.

Reply:
I really hope that fixes the doc build error, too. I can't easily find what is wrong with it.

Note that the code does not actually execute in the tutorial, so it doesn't show the plots. That's too bad because it looks like it would be usef : / Not sure how to fix that. But you could include that in the main function documentation instead of the tutorial instead of / in addition to the generic examples.

Reply:
Looks like doc build is OK!